content/deprecated-components/inline-autocomplete.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Inline autocomplete
```

Comment:
This has been moved to Primer recipes and probably doesn't need to be ported over https://ui.githubapp.com/storybook/?path=/story/others-components-forms-inlineautocomplete--default

Reply:
Is it possible someone is using this component in dotcom and needs to reference the deprecated docs?